Dockerfiles/agent/s6-services/trace/run

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/execlineb -P
+
+if { /initlog.sh "starting trace-agent" }
+fdmove -c 2 1
+trace-agent --ddconfig=/etc/datadog-agent/datadog.yaml
```

Comment:
`--ddconfig` is deprecated. Please use `--config`. See https://github.com/DataDog/datadog-trace-agent/pull/359.

Reply:
changed, thanks!